operations/experimental/enable-nfs-broker-backup.yml

```diff
@@ -0,0 +1,14 @@
+---
+  - type: replace
+    path: /instance_groups/name=backup-restore/jobs/-
+    value:
+      name: nfsbroker-bbr
+      release: nfs-volume
+      properties:
+        nfsbroker:
+          db_driver: ((nfsbroker-db-driver))
```

Comment:
`db_driver` is always set to `mysql` in the `enable-volume-service-nfs.yml` ops-file. Let's assume mysql here as well, and we can add a separate ops-file for using postgres if we want.